Description: Append CXXFLAGS instead of overwriting them.
 Otherwise the dpkg-buildflags are ignored.
Author: Felix Geyer <fgeyer@debian.org>

--- blobby-1.0~rc4.orig/src/CMakeLists.txt
+++ blobby-1.0~rc4/src/CMakeLists.txt
@@ -3,8 +3,8 @@ add_subdirectory(raknet)
 add_subdirectory(blobnet)
 
 add_definitions(-DTIXML_USE_STL)
-set(CMAKE_CXX_FLAGS "-std=c++11")
-set(CMAKE_CXX_FLAGS "-Wall")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
 include_directories(.)
 
 set(common_SRC
